java/launcher/src/main/java/com/lightstep/launcher/client/Client.java

```diff
@@ -0,0 +1,80 @@
+package com.lightstep.launcher.client;
+
+import com.lightstep.opentelemetry.common.VariablesConverter;
+import com.lightstep.opentelemetry.launcher.OpenTelemetryConfiguration;
+import io.grpc.Context;
+import io.opentelemetry.OpenTelemetry;
+import io.opentelemetry.exporters.otlp.OtlpGrpcSpanExporter;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.trace.export.SimpleSpanProcessor;
+import io.opentelemetry.trace.Span;
+import io.opentelemetry.trace.Span.Kind;
+import io.opentelemetry.trace.Tracer;
+import io.opentelemetry.trace.TracingContextUtils;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+
+public class Client {
+  public static void main(String[] args) {
+    String targetURL = System.getenv("TARGET_URL");
+    if (targetURL == null || targetURL.length() == 0) {
+      targetURL = "http://127.0.0.1:8084";
+    }
+
+    final String spanEndpoint = System.getenv("OTEL_EXPORTER_OTLP_SPAN_ENDPOINT");
+    final String lsToken = System.getenv("LS_ACCESS_TOKEN");
+    final String serviceName = System.getenv("LS_SERVICE_NAME");
+    final String serviceVersion = System.getenv("LS_SERVICE_VERSION");
+
+    OtlpGrpcSpanExporter exporter = OpenTelemetryConfiguration.newBuilder()
```

Comment:
why do you have to do this manually (doesn't the launcher pull from the environment variables?)

Reply:
was added to show explicit configuration.
removed